clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -253,397 +286,114 @@
     private final UnknownServerException unknownServerException = new UnknownServerException("secret");
```

Comment:
Actually a comment for line 283: If the intent is that this test covers all requests and responses, maybe we should say that in the class javadoc.